### ISSUE TITLE
Upgrade to Terracotta 3.2.0.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [byzhang/terrastore#25]
```
Upgrade to the latest Terracotta 3.2.0.
```

Original issue reported on code.google.com by `sergio.b...@gmail.com` on 14 Jan 2010 at 9:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `sergio.b...@gmail.com` on 14 Jan 2010 at 9:27
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Upgraded to terracotta 3.2.0, and used patched tim-concurrent-collections-1.3.0
version (at least for now).
```

Original comment by `sergio.b...@gmail.com` on 15 Jan 2010 at 6:03
- Changed state: **Fixed**
